docs/ha.md

```diff
@@ -1,77 +1,97 @@
-# Highly available controlplanes
+# Highly available control planes
```

Comment:
WDYT, does it make sense to emphasize in the title that this is for hosted control planes?